### PR TITLE
Update NPM modules

### PR DESCRIPTION
**Pull request by mramato**
No code changes needed except for some new eslint failures due to double line breaks and I had to fix one test suite which was using beforeAll instead of beforeEach.

@lilleyse this has the one bad failing test we just talked about, hopefully it's an easy fix.

### CODE REVIEW

File: specs/lib/removeUnusedVerticesSpec.js [hunk 1]
```diff
@@ -1,592 +1,591 @@
-'use strict';
```
Comment (mramato) [inline specs/lib/removeUnusedVerticesSpec.js]: The line endings in this file were wrong, which is why changing updated every line in the file.  All I did was remove an extra line break, no need to review.

**Comment by lilleyse** [comment]
Fixed the test. I'll merge once CI passes.

**Comment by mramato** [comment]
Thanks!

**Comment by emackey** [comment]
Will this get merged down into 2.0 and 2.0-cesium?

**Comment by mramato** [comment]
> Will this get merged down into 2.0 and 2.0-cesium?

I don't see why not (though the changes are super trivial and shouldn't effect either at all)

**Pull request merged**